lib/util/git/index.ts

```diff
@@ -469,7 +521,7 @@ export async function getFileList(): Promise<string[]> {
   const submodules = await getSubmodules();
   let files: string;
   try {
-    files = await git.raw(['ls-tree', '-r', branch]);
+    files = await gitRetry(() => git.raw(['ls-tree', '-r', branch]));
```

Comment:
I think this is also a pure offline command, but not fully sure

Reply:
According to https://git-scm.com/docs/git-ls-tree, the `ls-tree` command looks to be offline.